client/my-sites/design-preview/index.js

```diff
@@ -32,10 +32,11 @@ import DesignMenu from 'blocks/design-menu';
 import { getSiteFragment } from 'lib/route/path';
 import { getCurrentLayoutFocus } from 'state/ui/layout-focus/selectors';
 import { setLayoutFocus } from 'state/ui/layout-focus/actions';
+import AsyncLoad from 'components/async-load';
 
 const debug = debugFactory( 'calypso:design-preview' );
 
-export default function designPreview( WebPreview ) {
+export default function designPreview() {
```

Comment:
Seems to be broken the same way as `url-preview`: exporting as HOC, importing as regular React class.